CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.2.0]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+feat: check for external ids in batchs when creating aliases.
+
```

Comment:
Curious why it went directly to 0.2.0 instead of 0.1.9?

Reply:
It's a feature, not a patch, I guess?.  We tend to be a little too conservative with our semantic version numbering, we should be pumping out new minor versions more frequently.

Reply:
🤦🏽  Right, glossed over that logic of "major-version.medium-version.patch" . Makes sense to bump it as a feature vs a patch